codegen/projections/white_label/lib/white_label/plugins/test_plugin.rb

```diff
@@ -1,12 +1,18 @@
 module WhiteLabel
   module Plugins
     class TestPlugin
+
+      TEST_CLASS_INTERCEPTOR = Class.new do
```

Comment:
Why is this a constant? I think you can just define a normal class here, and then call `config.interceptors = TestInterceptor.new`

Reply:
I wanted to avoid using `any_instance_of` - this lets the test be a bit more specific.